packages/web-components/fast-components/src/horizontal-scroll/horizontal-scroll.styles.ts

```diff
@@ -0,0 +1,121 @@
+import { css } from "@microsoft/fast-element";
+import { DirectionalStyleSheetBehavior } from "@microsoft/fast-foundation";
+
+const ltrActionsStyles = css`
+    .scroll-prev {
+        right: auto;
+        left: 0;
+    }
+
+    .scroll.scroll-next:before {
+        left: auto;
+        right: 0;
+        background: linear-gradient(to right, transparent, var(--scroll-fade-next));
+    }
+
+    .scroll-next .scroll-action {
+        left: auto;
+        right: 0;
+        transform: translate(50%, -50%);
+    }
+`;
+
+const rtlActionsStyles = css`
+    .scroll-next {
+        right: auto;
+        left: 0;
+    }
+
+    .scroll.scroll-next:before {
+        left: auto;
+        right: 0;
+        background: linear-gradient(to right, var(--scroll-fade-next), transparent);
+    }
+
+    .scroll.scroll-prev:before {
+        background: linear-gradient(to right, transparent, var(--scroll-fade-previous));
+    }
+
+    .scroll-prev .scroll-action {
+        left: auto;
+        right: 0;
+        transform: translate(50%, -50%);
+    }
+`;
+
+/**
+ * Styles used for the flipper container and gradient fade
+ * @public
+ */
+export const ActionsStyles = css`
+    .scroll-area {
+        position: relative;
+    }
+
+    div.scroll-view {
```

Comment:
Does this need to be qualified?

Reply:
I needed this when I load these styles using $fastController.addStyles. Seems like it is prepending rather than appending styles maybe? They weren't overriding mobile styles when needed.